### ISSUE TITLE
Staging Review finding: Broken button for Veterans Crisis Line modal

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#104793]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Ask VA Integration
**Product:** Ask VA
**Feature:** Ask Va

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/ask-va/introduction](https://staging.va.gov/contact-us/ask-va/introduction) under the heading "Only use Ask VA for non-urgent needs," there's a button labeled "Connect with the Veterans Crisis Line." Currently, clicking that button doesn't do anything.

Looking under the hood a bit, I see that the crisis line button is coded as:
```
<va-button secondary="" text="Connect with the Veterans Crisis Line" href="https://www.veteranscrisisline.net/" class="hydrated"></va-button>
```

This is not correct usage of the [va-button component](https://design.va.gov/components/button/). Buttons do not navigate and do not support the `href` attribute.

Marking this one as a launch blocker because even though it doesn't imapct the user flow through the product it does have potential health and safety consequences.

<details><summary>Link, screenshot or steps to recreate</summary>

!["Only use Ask VA for non-urgent needs" section, with instructions for what to do for urgent needs. A secondary button is labeled "Connect with the Veterans Crisis Line."](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/62304-Brian-1.png?raw=true)

</details>

## Recommended action

In most cases, VA practices material honesty: the right element is used for the right job, and it looks like what users expect that element to look like. That means something that navigates is coded as a link and looks like a link, and something that performs an action on the page is coded as a button and looks like a button.

For this element, you could either code it as a button and have it open the [va-crisis-line-modal](https://design.va.gov/components/modal/crisis-line-modal), or you can code it as a link and have it navigate to veteranscrisisline.net. Something like [va-link-action](https://design.va.gov/components/link/action) would be great here.

## References

- **Accessibility Defect Severity:** [0: Potentially Harmful. Must be fixed immediately.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-0)
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by arielma-tw** [comment]
Issue has been resolved in https://github.com/department-of-veterans-affairs/ask-va/issues/1661
